kind-config.yaml

```diff
@@ -2,7 +2,7 @@ kind: Cluster
 apiVersion: kind.x-k8s.io/v1alpha4
 nodes:
   - role: control-plane
-    image: kindest/node:v1.16.15@sha256:a89c771f7de234e6547d43695c7ab047809ffc71a0c3b65aa54eda051c45ed20
+    image: quay.io/infinispan-kindest/node:v.17.17
```

Comment:
Image uploaded. Please update to quay.io/infinispan-test/kindest-node:v1.17.17

Reply:
I change it to quay.io/infinispan-test/kindest-node:v1.17.17